src/python/pants/jvm/resolve/coursier_setup.py

```diff
@@ -125,14 +139,22 @@ class Coursier:
     _digest: Digest
 
     bin_dir: ClassVar[str] = "__coursier"
-    wrapper_script: ClassVar[str] = f"{bin_dir}/coursier_wrapper_script.sh"
+    fetch_wrapper_script: ClassVar[str] = f"{bin_dir}/coursier_fetch_wrapper_script.sh"
     post_processing_script: ClassVar[str] = f"{bin_dir}/coursier_post_processing_script.py"
+    post_process_stderr: ClassVar[str] = f"{bin_dir}/coursier_post_process_stderr.py"
     cache_name: ClassVar[str] = "coursier"
     cache_dir: ClassVar[str] = ".cache"
     working_directory_placeholder: ClassVar[str] = "___COURSIER_WORKING_DIRECTORY___"
 
     def args(self, args: Iterable[str], *, wrapper: Iterable[str] = ()) -> tuple[str, ...]:
-        return tuple((*wrapper, os.path.join(self.bin_dir, self.coursier.exe), *args))
+        return tuple(
```

Comment:
I don't think we need the outer `tuple()` call - it's already a tuple. Ack that it was already that way

Reply:
Good catch, thanks